modules/cart/src/Plugin/Block/CartBlock.php

```diff
@@ -125,9 +158,14 @@ public function build() {
     });
 
     $count = 0;
-    $cart_views = [];
+    $content = [];
     if (!empty($carts)) {
-      $cart_views = $this->getCartViews($carts);
+      if ($this->configuration['view']) {
```

Comment:
This is causing the error, because no `!empty()` check, which also means it needs a default value of `NULL`.